packages/spacecat-shared-data-access/src/dto/api-key.js

```diff
@@ -0,0 +1,53 @@
+/*
+ * Copyright 2024 Adobe. All rights reserved.
+ * This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License. You may obtain a copy
+ * of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under
+ * the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ * OF ANY KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+import { createApiKey } from '../models/api-key.js';
+
+export const ApiKeyDto = {
+
+  /**
+     * Converts an ApiKey object into a DynamoDB item.
+     * @param apiKey
+     * @returns {{createdAt: *, name: *, imsUserId, scopes, revokedAt, key: *,
+     *          imsOrgId: *, expiresAt: *}}
+     */
+  toDynamoItem: (apiKey) => ({
+    key: apiKey.getKey(),
```

Comment:
Should this be `hashedKey` (or similar) so it does not get misinterpreted as the actual key?